### ISSUE TITLE
[BUG]: Can't have space in column alias field (Query Builder)

### ISSUE THREAD

**Issue by JoeyOverby** [phalcon/cphalcon#14535]
**Describe the bug**
I'm unable to have a field in a query get returned that has a space in it. For example if I wanted the field 'lastName' to display as 'Last Name'

I've tried putting quotes, escaping quotes, etc in that key, but none of it will take effect. 

`Phalcon\Mvc\Model\Exception : Scanning error before 'First Name], Peo...' when parsing: SELECT People.firstName AS [First Name], People.lastName AS [Last Name] FROM [BMS\Test\Database\Models\People] (110)`

**To Reproduce**

Simple use any Phalcon Model class you have to get the query builder. 

> Provide minimal script to reproduce the issue

```php

/** @var \Phalcon\Mvc\Model $peopleModel */
            $peopleModel = new Models\People();
            
            $models = [
                0 => 'Models\People',
            ];
    
            $columns = [
                'First Name' => 'People.firstName',
                'Last Name'  => 'People.lastName',
            ];
    
            $builder = $peopleModel->getModelsManager()->createBuilder();
    
            $builder->from(Models\People::class);
    
            $builder->columns($columns);
    
            $query  = $builder->getQuery();
            $result = $query->execute();
// paste output here
```

**Expected behavior**

I would expect that I could in fact re-name a column to have a space in it, like I can in SQL. 
SELECT `People`.`firstName` AS `First Name`, 

**Details**
 - Phalcon version: (`php --ri phalcon`)
php --ri phalcon

phalcon

Web framework delivered as a C-extension for PHP
phalcon => enabled
Author => Phalcon Team and contributors
Version => 3.4.1
Build Date => Nov 12 2018 13:54:28
Powered by Zephir => Version 0.10.10-d1b4cc68d9

Directive => Local Value => Master Value
phalcon.db.escape_identifiers => On => On
phalcon.db.force_casting => Off => Off
phalcon.orm.events => On => On
phalcon.orm.virtual_foreign_keys => On => On
phalcon.orm.column_renaming => On => On
phalcon.orm.not_null_validations => On => On
phalcon.orm.exception_on_failed_save => Off => Off
phalcon.orm.enable_literals => On => On
phalcon.orm.late_state_binding => Off => Off
phalcon.orm.enable_implicit_joins => On => On
phalcon.orm.cast_on_hydrate => Off => Off
phalcon.orm.ignore_unknown_columns => Off => Off
phalcon.orm.update_snapshot_on_save => On => On
phalcon.orm.disable_assign_setters => Off => Off

 - PHP Version: (`php -v`)
PHP 7.2.18 (cli) (built: May  2 2019 13:03:01) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Xdebug v2.6.1, Copyright (c) 2002-2018, by Derick Rethans
    with Zend OPcache v7.2.18, Copyright (c) 1999-2018, by Zend Technologies

 - Operating System:
Mac OS

**Additional context**

**Comment by niden** [comment]
This seems to be an issue with the PHQL parser. Assigning it to @sergeyklay to work his magic.

**Comment by JoeyOverby** [comment]
Thank you, this is a huge issue for me right now!

**Comment by Jeckerson** [comment]
Try with double quotes

```php
$columns = [
    '"First Name"' => 'People.firstName',
    '"Last Name"'  => 'People.lastName',
];
```

**Comment by JoeyOverby** [comment]
It's a good idea, but I have tried and it doesn't work. Tried escaping
quotes inside of quotes, single and double quote combinations, etc.

**Comment by Jeckerson** [comment]
Last try with `new \Phalcon\Db\RawValue("First name");` ?

**Comment by JoeyOverby** [comment]
@Jeckerson,
  You can't use that type as the array key for the columns array as it isn't a string, or an integer. 
![image](https://user-images.githubusercontent.com/38049004/69063446-bce68a80-09d9-11ea-90e7-3a8fa9625503.png)

And trying to use that to wrap/convert the string doesn't work as the string value (through __string or getValue(), still equals the string with a space) 

```php
    /**
     * Verification showing that the new RawValue function doesn't wrap or escape the string. 
     * @test
     */
    public function testUsingRawValue() : void{
        $something = new RawValue("First name");
        
        self::assertTrue($something->__toString() === "First name", "Not equal to the string representation");
        self::assertTrue($something->getValue() === "First name", "Not equal to the string representation");
        
    }
```

**Comment by sergeyklay** [comment]
Implemented in 4.1.x branch
